app/src/main/java/org/piwigo/bg/AlbumService.java

```diff
@@ -49,20 +50,18 @@ protected void onHandleIntent(Intent intent) {
             @Override
             public void onResponse(Call<AddCategoryResponse> call, Response<AddCategoryResponse> response) {
                 if (response.raw().code() == 200) {
-                    if (response.body().stat.equals("ok")) {
-                        // TODO: make text localizable
-                        Toast.makeText(getApplicationContext(), "Created album: " + catName + " !", Toast.LENGTH_LONG).show();
-                    }
+                    if (response.body().stat.equals("ok"))
+                        Toast.makeText(getApplicationContext(), R.string.create_album_success, Toast.LENGTH_LONG).show();
                     else
-                        Toast.makeText(getApplicationContext(), "Unable to create a new album..", Toast.LENGTH_LONG).show();
+                        Toast.makeText(getApplicationContext(), R.string.create_album_error, Toast.LENGTH_LONG).show();
                 }
                 else
-                    Toast.makeText(getApplicationContext(), "Error while creating album..", Toast.LENGTH_LONG).show();
+                    Toast.makeText(getApplicationContext(), R.string.create_album_error, Toast.LENGTH_LONG).show();
             }
 
             @Override
             public void onFailure(Call<AddCategoryResponse> call, Throwable t) {
-                // TODO: do something
+                t.printStackTrace();
```

Comment:
nobody will see this printing, I fear.

Reply:
Please also use Log...

Reply:
Feel free to click on 'Resolve conversation' yourself, after you fixed something